### PR TITLE
elaborate versioning strategy

### LINKED ISSUE #204: 1.0.0 and versioning article
declare `1.0.0` and elaborate the versioning strategy for TROLIE 1.0

### PR DESCRIPTION
**Pull request by catkins-miso**
closes #204

### CODE REVIEW

**Comment by catkins-miso** [comment]
change language to "supporting *incremental* conformance" as opposed to *partial*

File: docs/decision-log/media-type-versioning.md [hunk 1]
```diff
@@ -5,35 +5,150 @@ parent: Decision Records
...
+fixes".
+
+For example, if a property is obviously required in an exchange, but it is not
+formally required in the spec due to an error of omission, we would make it
+required and increment the `patch` version, i.e., `1.1.1` &rarr; `1.1.2`.
+Another example of a "bug fix" would be a spelling error or a property rename
+that results in better uniformity without impacting semantics.  The logic here
+is that these are bugs in the specification and implementations would not have
+committed to that error. The point is the spec is meant to serve the
+implementors, not the other way around; interop is the goal, not conformance.
```
Comment (catkins-miso) [inline docs/decision-log/media-type-versioning.md]:
```suggestion
implementors, not the other way around; interop is the goal, not blind conformance.
```

File: docs/decision-log/media-type-versioning.md [hunk 2]
```diff
@@ -5,35 +5,150 @@ parent: Decision Records
...
-Pervasive use of media types allow for different representations to be provided
-for the same resource. This allows the spec to support anticipated use cases
-requiring different information or performance characteristics. If new
-representations are identified later, these can be added to the spec without
-breaking existing clients.
+Pervasive use of versioned media types allow for different representations to be
+provided for the same resource. This allows the TROLIE 1.0 spec to be updated in
+the future efforts without breaking existing implementations. It also supports
+the Conformance program by providing granular delineation of the functionality
+supported by different implementations.
```
Comment (catkins-miso) [inline docs/decision-log/media-type-versioning.md]:
```suggestion
supported by different implementations, supporting incremental conformance.
```

File: docs/decision-log/media-type-versioning.md [hunk 3]
```diff
@@ -5,35 +5,150 @@ parent: Decision Records
...
+Clearinghouse-to-Clearinghouse communications. Accordingly, we must understand
+breaking changes from all these perspectives. It is anticipated that some TROLIE
+implementations will partially adopt the TROLIE 1.0 specification. For
+example, a vendor may only choose to implement the Forecasting operations, or
+two adjacent Transmission Owners might choose to only share seasonal ratings.
+
+Enabling this kind of a la carte use of the specification by the community is
+desirable, and a [Conformance program](https://github.com/trolie/conformance) to
+provide clarity on what is implemented by various vendors was part of the
+project's original charter.
```
Comment (catkins-miso) [inline docs/decision-log/media-type-versioning.md]:
```suggestion
project's original charter, with the goal of supporting incremental conformance; not every feature will be supported in every implementation, and incremental conformance is about ensuring that parts that are implemented have an unambiguous specification that can be verified with automated tests.
```

File: docs/decision-log/media-type-versioning.md [hunk 4]
```diff
@@ -5,35 +5,150 @@ parent: Decision Records
...
-* Status: `accepted`
+* Status: `committed`
 
 ## Context
 
-The TROLIE project needs a plan for updating the specification after the
-1.0 scope is complete, that is, after the spec is published as 1.0 and clients
-and servers are deployed that are dependent on that version of the spec.
+The primary mandate of the TROLIE project is defined by its
+[Scope](https://github.com/trolie/spec/blob/1.0.0-wip/Scope.md).  Paramount to
```
Comment (getorymckeag) [inline docs/decision-log/media-type-versioning.md]: You're going to want to make these links relative so they don't die when you change the branch name.

**Review by getorymckeag — CHANGES_REQUESTED** [review]
This is pretty well thought through.  If you have a chance, I say just fix the links.

File: docs/decision-log/media-type-versioning.md (hunk 4 above)
Comment (catkins-miso) [inline docs/decision-log/media-type-versioning.md]:
```suggestion
[Scope](https://github.com/trolie/spec/tree/HEAD/Scope.md).  Paramount to
```

File: docs/decision-log/media-type-versioning.md [hunk 5]
```diff
@@ -5,35 +5,150 @@ parent: Decision Records
...
 ## Context
 
-The TROLIE project needs a plan for updating the specification after the
-1.0 scope is complete, that is, after the spec is published as 1.0 and clients
-and servers are deployed that are dependent on that version of the spec.
+The primary mandate of the TROLIE project is defined by its
+[Scope](https://github.com/trolie/spec/tree/HEAD/Scope.md).  Paramount to
+meeting that mandate is defining the [TROLIE 1.0 API
+specification](https://trolie.energy/spec), yet this is a [community
+specification](https://github.com/trolie/spec/blob/1.0.0-wip/Community_Specification_License.md)
```
Comment (getorymckeag) [inline docs/decision-log/media-type-versioning.md]:
```suggestion
specification](https://github.com/trolie/spec/blob/HEAD/Community_Specification_License.md)
```

**Review by getorymckeag — APPROVED** [review]

**Pull request merged**